### PR TITLE
added "First-time contributors" section

### PR DESCRIPTION
**Pull request by dunkOnIT**
Just a small change - figured that as I'm learning things I may as well write them down for others newbies. I think it makes sense that every project should have a first-time contributors section (Github is hard enough already lol). 

However, there is a balance here between providing information ourselves and pointing others to tutorials. My reasoning for including this content in particular is that despite consulting a _number_ of Git tutorials and Open Source guides, I wasn't able to find a concise explanation of the standard process for actually making a pull request.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,6 +4,19 @@ Pull requests are welcome for any part of this project. Please feel free to send
...
 
+## First-Time Contributors
+
+If this is your first time contributing to a project on Github, these are the steps you need to take to contribute code:
+
+1. Create a fork of `cubing.js` 
+2. Install [Git](https://git-scm.com/downloads)
+3. Clone your forked version of `cubing.js` to your computer
+4. Create a new branch in your cloned project, and make the changes you would like to make 
+5. Push your changes to your forked cubing.js repo
```
Comment (lgarron) [inline CONTRIBUTING.md:15]:
```suggestion
5. Push your changes to your forked `cubing.js` repo
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,6 +4,19 @@ Pull requests are welcome for any part of this project. Please feel free to send
 
 If you have any thoughts about bigger contributions, please consider [filing an issue](https://github.com/cubing/cubing.js/issues/new) to discuss it before sending a pull request. (Since we're still [below version 1.0](https://semver.org/#spec-item-4), some relevant design choices are still not settled or written down yet.)
 
+## First-Time Contributors
+
+If this is your first time contributing to a project on Github, these are the steps you need to take to contribute code:
+
+1. Create a fork of `cubing.js`
```
Comment (lgarron) [inline CONTRIBUTING.md:11]: Could I ask you to put a period (`.`) at the end of every bullet?

There's nothing wrong with leaving it out, but that will make it more consistent with the rest of the project.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -4,6 +4,19 @@ Pull requests are welcome for any part of this project. Please feel free to send
...
+If this is your first time contributing to a project on Github, these are the steps you need to take to contribute code:
+
+1. Create a fork of `cubing.js` 
+2. Install [Git](https://git-scm.com/downloads)
+3. Clone your forked version of `cubing.js` to your computer
+4. Create a new branch in your cloned project, and make the changes you would like to make 
+5. Push your changes to your forked cubing.js repo
+6. Create a pull request in `cubing.js`, with your new branch from your forked repo as the source.
+
+Youtube has many [beginners guides](https://www.youtube.com/watch?v=RGOj5yH7evk) to Git and Github if you would like more information.
```
Comment (lgarron) [inline CONTRIBUTING.md:18]: This looks like a nice find!

```suggestion
Youtube has many guides to Git and Github if you would like more information. If you want a thorough overview, consider [Git and GitHub for Beginners - Crash Course](https://www.youtube.com/watch?v=RGOj5yH7evk).
```

**Review by lgarron — COMMENTED** [review]
Thanks, this is a nice way of laying out how to contribute!

File: CONTRIBUTING.md (hunk 2 above)
Comment (TydVirTaal) [reply CONTRIBUTING.md]: Periods added per request!

**Pull request merged**